File: app/src/main/java/com/example/todolist_ramkumartextiles/services/MessagingService.java
package com.example.todolist_ramkumartextiles.services;

import com.example.todolist_ramkumartextiles.helpers.NotificationHelper;
import com.google.firebase.messaging.FirebaseMessagingService;
import com.google.firebase.messaging.RemoteMessage;

public class MessagingService extends FirebaseMessagingService
{
    @Override
    public void onMessageReceived(RemoteMessage remoteMessage) {
        super.onMessageReceived(remoteMessage);

        if(remoteMessage.getNotification() != null)
        {
            String title = remoteMessage.getNotification().getTitle();
            String body = remoteMessage.getNotification().getBody();
            NotificationHelper.displayNotification(getApplicationContext(),title,body);
        }
    }
}

File: app/src/main/java/com/example/todolist_ramkumartextiles/services/MessagingService.kt
package com.example.todolist_ramkumartextiles.services

import com.example.todolist_ramkumartextiles.helpers.NotificationHelper
import com.google.firebase.messaging.FirebaseMessagingService
import com.google.firebase.messaging.RemoteMessage

class MessagingService : FirebaseMessagingService() {
    override fun onMessageReceived(remoteMessage: RemoteMessage?) {
        super.onMessageReceived(remoteMessage)

        if (remoteMessage!!.notification != null) {
            val title = remoteMessage.notification!!.title
            val body = remoteMessage.notification!!.body
            NotificationHelper.displayNotification(applicationContext, title!!, body!!)
        }
    }
}
